UI/CMakeLists.txt

```diff
@@ -170,6 +168,16 @@ if(BROWSER_AVAILABLE_INTERNAL)
 	endif()
 endif()
 
+if(PEERTUBE_ENABLED)
+	add_definitions(-DPEERTUBE_ENABLED)
```

Comment:
Don't use `add_definitions`, as it adds compile flags to all(!) targets. I guess in your case you need to add a check to `rtmp-services` to check for the enable switch and then do 

```
    target_compile_definitions(rtmp-services
        PRIVATE PEERTUBE_ENABLED
```

and repeat the same in this file with the `obs` target.